tests/jballerina-unit-test/src/test/resources/test-src/reachability-analysis/narrowing_with_if_without_else_not_completed_normally_test.bal

```diff
@@ -428,3 +428,20 @@ function testTypeNarrowingWithEqualityInCondition2() {
 
     int j = i;
 }
+
+public function testReachabilityWithWorker1() returns error? {
```

Comment:
```suggestion
public function testTypeOfCapturedTypeNarrowedVariableInWorker() returns error? {
```

Reply:
Actually, this happens even without the return statement, right? So doesn't quite have anything to do with not returning normally.

I think we should move this to a separate test (worker tests maybe or if we have separate tests specifically for adding the block statement for if blocks without else blocks - cc @lasinicl) and change the name accordingly.

Reply:
Let's also add a test like

```ballerina
function fn() {
    boolean b = true;
    
    if b {
        
    }

    worker w1 {

    }
}
```